int32 lineNum
bool isStraight
geometry_msgs/Point[] points
float64[] poseWeights
